### PR TITLE
Changed references to RUN_ENV to DB_TYPE

### PR DESCRIPTION
**Pull request by slloydrob**
Changed references from RUN_ENV to DB_TYPE per the decision made in class.

### CODE REVIEW

File: backend/db/database.go [hunk 1]
```diff
@@ -26,11 +26,11 @@ type Database interface {
 
 func NewDatabaseFromEnv() (Database, error) {
 	if connection == nil {
-		runningEnv, wasSet := os.LookupEnv("RUN_ENV")
+		runningEnv, wasSet := os.LookupEnv("DB_TYPE")
```
Comment (ianjm24) [inline backend/db/database.go:29]: Just to be consistent
```suggestion
		dbType, wasSet := os.LookupEnv("DB_TYPE")
```

File: backend/db/database.go [hunk 2]
```diff
@@ -26,11 +26,11 @@ type Database interface {
...
 
 		if !wasSet || len(runningEnv) == 0 {
```
Comment (ianjm24) [inline backend/db/database.go:31]:
```suggestion
		if !wasSet || len(dbType) == 0 {
```

File: backend/db/database.go [hunk 3]
```diff
@@ -26,11 +26,11 @@ type Database interface {
...
-			log.Error("Environment variable RUN_ENV was not set correctly")
-			return nil, errors.New("RUN_ENV not set")
+			log.Error("Environment variable DB_TYPE was not set correctly")
+			return nil, errors.New("DB_TYPE not set")
 		}
 
 		switch runningEnv {
```
Comment (ianjm24) [inline backend/db/database.go:36]:
```suggestion
		switch dbType {
```

File: backend/db/database.go [hunk 4]
```diff
@@ -42,8 +42,8 @@ func NewDatabaseFromEnv() (Database, error) {
 			connection = &Mongo{}
 
 		default:
-			log.Error("Unknown RUN_ENV set %v", runningEnv)
-			return nil, errors.New("unknown RUN_ENV")
+			log.Error("Unknown DB_TYPE set %v", runningEnv)
```
Comment (ianjm24) [inline backend/db/database.go:45]:
```suggestion
			log.Error("Unknown DB_TYPE set %v", dbType)
```

**Review by ianjm24 — CHANGES_REQUESTED** [review]

**Review by ianjm24 — APPROVED** [review]

**Review by jebidiah252 — APPROVED** [review]
These changes look great! Thank you for making these updates!

**Pull request merged**